src/components/app/app.js

```diff
@@ -1,6 +1,23 @@
 import angular from "angular";
 import template from "./app.html";
 
-export const appModule = angular.module("appModule", []).component("app", {
-  template
-}).name;
+import ngMaterial from "angular-material";
+
+import { transactionDialogModule } from "../transaction-dialog/transaction-dialog";
+
+export class AppController {
+  constructor() {
+    this.transactionDialogOpen;
```

Comment:
why do you need this?

Reply:
if I removed this, dialog will be opened right after page loading

Reply:
sorry, yes. I it seems I don't need this. no problem anymore without `this.transactionDialogOpen;` in constructor